src/main/java/com/gregtechceu/gtceu/integration/emi/recipe/Ae2PatternTerminalHandler.java

```diff
@@ -37,7 +37,7 @@ public EmiPlayerInventory getInventory(AbstractContainerScreen<T> screen) {
 
     @Override
     public boolean supportsRecipe(EmiRecipe recipe) {
-        return recipe instanceof GTEmiRecipe;
+        return recipe instanceof GTEmiRecipe | recipe instanceof MultiblockInfoEmiRecipe;
```

Comment:
Use the short-circuiting operator `||`, also you forgot to import the class??

Reply:
Yep, forgot to import the class, my bad. `|` was also a typo lol